docs/data/product/dea-geometric-median-and-median-absolute-deviation-landsat/_details.md

```diff
@@ -215,7 +215,7 @@ $$
 \end{align*}
 $$
 
-In GeoMAD, the MADs are calculated from the same ten bands used in the geometric median, therefore $p=10$. The result of $\lVert \mathbf{x}^{(t)} - \mathbf{m} \rVert_{\mathbb{R}^p}$ is a positive scalar, so $\text{EMAD}_\text{GeoMAD}$ is a positive scalar number. As in the geometric median, $N$ is dependent on the number of satellite flyovers particular to that pixel.
+In GeoMAD, the MADs are calculated from the same six bands used in the geometric median, therefore $p=6$. The result of $\lVert \mathbf{x}^{(t)} - \mathbf{m} \rVert_{\mathbb{R}^p}$ is a positive scalar, so $\text{EMAD}_\text{GeoMAD}$ is a positive scalar number. As in the geometric median, $N$ is dependent on the number of satellite flyovers particular to that pixel.
```

Comment:
```suggestion
In GeoMAD, the MADs are calculated from the same 6 bands used in the geometric median, therefore $p=6$. The result of $\lVert \mathbf{x}^{(t)} - \mathbf{m} \rVert_{\mathbb{R}^p}$ is a positive scalar, so $\text{EMAD}_\text{GeoMAD}$ is a positive scalar number. As in the geometric median, $N$ is dependent on the number of satellite flyovers particular to that pixel.
```

In scientific writing, I now try to use numerals instead.

Reply:
There's a few different recommended approaches here I think - I was always taught to spell out numbers less than 10, and use numerals for anything greater than that ([AP/APA style here](https://www.masterclass.com/articles/when-to-spell-out-numbers-explained)) - I've seen spelling them out below 100 recommended too.

Reply:
I did some research and actually scientific writing uses one to nine in worded form, as you mentioned. I read something recently that gave me the wrong idea.